.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You shouldn't add IDE-related files to the repo, because other devs can use different setups.